src/geofirestore.ts

```diff
@@ -96,21 +96,30 @@ export class GeoFirestore {
    *
    * If any provided key already exists in this GeoFirestore, it will be overwritten with the new location value.
    *
+   * @param writeType Specifies the type of write (set || update).
    * @param keyOrDocuments The key representing the document to add or an object of $key - document pairs.
    * @param document The document to be added to the GeoFirestore.
    * @param customKey The key of the document to use as the location. Otherwise we default to `coordinates`.
    * @returns A promise that is fulfilled when the write is complete.
    */
-  public set(keyOrDocuments: string | any, document?: any, customKey?: string): Promise<any> {
+  private write(writeType: string, keyOrDocuments: string | any, document?: any, customKey?: string): Promise<any> {
```

Comment:
Also private functions and public functions are kept separate, and in alphabetical order. I'm not sure if this is a change needed anywhere else, but please keep that in mind (and update the code to reflect that).

Reply:
Good call! 
Corrected in : https://github.com/geofirestore/geofirestore-js/pull/53/commits/716eda9ed356cb1a078f6fc9fe1385b37973c6af